### ISSUE TITLE
Name spelling

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#5084]
**Name spelling**

I needed a fungal example and I took the species where I co-authored a comb. nov. Looks like my last name is one letter short (but backbone display ok) - is this Species Fungorum's typo?

-----

Github user: @dschigel
User: [See in registry](https://www.gbif.org/api/feedback/user/3e206d192ee4f80e37e86eee0327c3fd:ffc9497647fd2e826d635f6c132009929e48170ab5afaffd1d9a5dc37812c76f15a53bc23603a5d54b4e72e756f8acb2c8a9368a7d47c5bb25855b42670e4393cc412e0261fb436fac0096b5f484866f08c08fe63d2b8450f8e74afe3fd418c7ee413b12c91092e78e56ee3233e63f06a44669eb62dc68afc4f4927b61a448b9) - [Send email](https://www.gbif.org/api/feedback/user/mailto/3e206d192ee4f80e37e86eee0327c3fd:ffc9497647fd2e826d635f6c132009929e48170ab5afaffd1d9a5dc37812c76f15a53bc23603a5d54b4e72e756f8acb2c8a9368a7d47c5bb25855b42670e4393cc412e0261fb436fac0096b5f484866f08c08fe63d2b8450f8e74afe3fd418c7ee413b12c91092e78e56ee3233e63f06a44669eb62dc68afc4f4927b61a448b9)
System: Chrome 120.0.0 / Windows 10.0.0
Referer: https://www.gbif.org/species/2542856
Window size: width 1638 - height 801
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-12-08T09:43:30.875Z',to:'2023-12-08T09:49:30.875Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-12-08T09:43:30.875Z',to:'2023-12-08T09:49:30.875Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by ManonGros** [comment]
@dschigel yes it looks it comes directly from Species Fungorum, see http://www.indexfungorum.org/Names/NamesRecord.asp?RecordID=500979

> in Niemelä, Kinnunen, Schige & Larsson, Karstenia 45(2): 79 (2005)

**Comment by dschigel** [comment]
Thanks for confirming the root cause; I have seen worse and Kew is recruiting a db manager for IF - maybe ?SF, too. I will wait for the post to be filled before attempting a fix
